execution/executor-benchmark/src/main.rs

```diff
@@ -345,13 +348,20 @@ where
                 Some(mix_per_phase[0].clone())
             };
 
+            let maybe_hotspot_probability =
+                if opt.hotspot_probability >= 0.5 && opt.hotspot_probability < 1.0 {
```

Comment:
I'd rather panic when it's out of range, instead of swallowing it.

Reply:
okay I see you assert down the stack.. but anyways, easier to read if asserted here.

Reply:
fixed